.gitignore

```diff
@@ -7,3 +7,4 @@ _vimrc_local.vim
 config/config.php
 test/coverage
 .vagrant
+composer.lock
```

Comment:
Don't ignore the composer.lock, this is an application no library and we want to lock down our dependencies exactly to always ensure being able to run it.